File: ui/closures/net.bluemind.todolist.ui.closure/src/main/javascript/net/bluemind/todolist/Messages.js
/* BEGIN LICENSE
 * Copyright © Blue Mind SAS, 2012-2016
 *
 * This file is part of BlueMind. BlueMind is a messaging and collaborative
 * solution.
 *
 * This program is free software; you can redistribute it and/or modify
 * it under the terms of either the GNU Affero General Public License as
 * published by the Free Software Foundation (version 3 of the License).
 *
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 *
 * See LICENSE.txt
 * END LICENSE
 */

goog.provide("net.bluemind.task.Messages");

net.bluemind.task.Messages.errorLoadingLists = function(msg) {
  /** @meaning task.todolists.loadingError */
  var MSG_ERROR = goog.getMsg('Error during Todo lists loading: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorLoading = function(msg) {
  /** @meaning generic.loadingError */
  var MSG_ERROR = goog.getMsg('Error during loading: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorCreate = function(msg) {
  /** @meaning generic.createError */
  var MSG_ERROR = goog.getMsg('Error during creation: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.successCreate = function() {
  /** @meaning task.create.success */
  var MSG_ERROR = goog.getMsg('Task created');
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorUpdate = function(msg) {
  /** @meaning generic.updateError */
  var MSG_ERROR = goog.getMsg('Error during update: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.successUpdate = function() {
  /** @meaning task.update.success */
  var MSG_ERROR = goog.getMsg('Task updated');
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorDelete = function(msg) {
  /** @meaning generic.deleteError */
  var MSG_ERROR = goog.getMsg('Error during delete: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.successDelete = function() {
  /** @meaning task.delete.success */
  var MSG_ERROR = goog.getMsg('Task deleted');
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorMove = function(msg) {
  /** @meaning task.move.error */
  var MSG_ERROR = goog.getMsg('Error during move: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.successMove = function() {
  /** @meaning task.move.success */
  var MSG_ERROR = goog.getMsg('Task moved');
  return MSG_ERROR;
}

net.bluemind.task.Messages.errorCopy = function(msg) {
  /** @meaning task.copy.error */
  var MSG_ERROR = goog.getMsg('Error during copy: {$message}', {
    message : msg
  });
  return MSG_ERROR;
}

net.bluemind.task.Messages.successCopy = function() {
  /** @meaning task.move.success */
  var MSG_ERROR = goog.getMsg('Task copied');
  return MSG_ERROR;
}

File: parent/ldap/net.bluemind.system.ldap.importation/src/net/bluemind/system/ldap/importation/internal/l10n/Messages.java
/* BEGIN LICENSE
  * Copyright © Blue Mind SAS, 2012-2016
  *
  * This file is part of BlueMind. BlueMind is a messaging and collaborative
  * solution.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of either the GNU Affero General Public License as
  * published by the Free Software Foundation (version 3 of the License).
  *
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  *
  * See LICENSE.txt
  * END LICENSE
  */
package net.bluemind.system.ldap.importation.internal.l10n;

import java.util.Locale;
import java.util.MissingResourceException;
import java.util.ResourceBundle;

public class Messages {
	private ResourceBundle bundle;

	public static Messages get(Locale locale) {
		Messages messages = new Messages();

		if (locale == null) {
			locale = new Locale("en");
		}

		try {
			messages.bundle = ResourceBundle.getBundle("OSGI-INF/l10n/messages", locale);
		} catch (MissingResourceException mre) {
			messages.bundle = ResourceBundle.getBundle("OSGI-INF/l10n/messages");
		}

		return messages;
	}

	public String invalidLoginDn() {
		return bundle.getString("invalidLoginDn");
	}

	public String invalidBaseDn() {
		return bundle.getString("invalidBaseDn");
	}

	public String nullLdapProtocol() {
		return bundle.getString("nullLdapProtocol");
	}

	public String invalidProtocol() {
		return bundle.getString("invalidProtocol");
	}

	public String invalidHostname() {
		return bundle.getString("invalidHostname");
	}

	public String invalidUserFilter() {
		return bundle.getString("invalidUserFilter");
	}

	public String invalidGroupFilter() {
		return bundle.getString("invalidGroupFilter");
	}
}
